CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * Support setting `maxNumberOfActiveVersions` when creating a `Configuration`. ([#1036](https://github.com/realm/realm-dart/pull/1036))
 * Add List.move extension method that moves an element from one index to another. Delegates to ManagedRealmList.move for managed lists. This allows notifications to correctly report moves, as opposed to reporting moves as deletes + inserts. ([#1037](https://github.com/realm/realm-dart/issues/1037))
 * Support setting `shouldDeleteIfMigrationNeeded` when creating a `Configuration.local`. ([#1049](https://github.com/realm/realm-dart/issues/1049))
+* If `syncErrorHandler` occurs with a new unknown error code then return `SyncSessionErrorCode.unknown`, `SyncConnectionErrorCode.unknown`, `SyncClientErrorCode.unknown` or `GeneralSyncErrorCode.unknown` instead of throwing exception. ([#1052](https://github.com/realm/realm-dart/pull/1052))
```

Comment:
What does it mean for a `syncErrorHandler to occur`.

I suggest we reword this to something like 
```suggestion
* Add `unknown` error code to all SyncErrors. `SyncSessionErrorCode.unknown`, `SyncConnectionErrorCode.unknown`, `SyncClientErrorCode.unknown` or `GeneralSyncErrorCode.unknown`. Use `unknown` error code instead of throwing a RealmError. ([#1052](https://github.com/realm/realm-dart/pull/1052))
```